cypress/support/pages/signIn.pageObject.js

```diff
@@ -5,30 +5,34 @@ class SignInPageObject extends PageObject {
 
   get emailField() {
     return cy.getByDataCy('email-sign-in');
+    return this.getByDataCy('email-sign-in');
   }
 
   get passwordField() {
     return cy.getByDataCy('password-sign-in');
+    return this.getByDataCy('password-sign-in');
   }
 
   get signInBtn() {
     return cy.getByDataCy('sign-in-btn');
+    return this.getByDataCy('sign-in-btn');
   }
 
   typeEmail(email) {
     this.emailField
       .type(email);
+        .type(email);
```

Comment:
There is a redundant type statement here. Remove the duplicate `.type(email);` to ensure the method is clear and concise.